client/app/widgets/Resources/index.jsx

```diff
@@ -110,28 +119,41 @@ export class Resources extends React.Component<Props, State> {
     });
   };
 
+  onClick = () => {
+    const { resources } = this.props;
+    this.setState((prevState: State) => ({
+      resourcesDisplayed: prevState.resourcesDisplayed + RESOURCES_PER_PAGE,
+      lastPage:
+        prevState.resourcesDisplayed + RESOURCES_PER_PAGE >= resources.length,
```

Comment:
We should write an integration test for this! More specifically, we should be testing clicking the LoadMore button in this component and being able to load more Resources on the page

Reply:
Should I add tests in Resources/__tests__/Resources.spec.jsx or create a separate file for the tests for LoadMoreButton component?

Reply:
Good question, so I think we should be prioritizing to test the LoadMoreButton component.